src/services/bookDownload/requests.js

```diff
@@ -12,6 +11,6 @@ export function* triggerDownload(bookIds) {
   };
 
   const api = yield put(getAPI());
-  const response = yield api.get(makeURI('/api/user_books/trigger_download', query, config.STORE_API_BASE_URL));
+  const response = yield api.post(makeURI('/api/user_books/trigger_download', {}, config.STORE_API_BASE_URL), stringify(query));
```

Comment:
오잉 요청 페이로드가 JSON이 아닌가요?

Reply:
페이로드 중 도서id 배열이 제대로 전달되지 않아 직렬화 했어요

Reply:
urlencoded가 아니라 JSON으로 줘야 하지 않을까요? (API 명세가 어디 있는지 모르겠네요)

```suggestion
  const response = yield api.post(makeURI('/api/user_books/trigger_download', {}, config.STORE_API_BASE_URL), query);
```

Reply:
http://api.dev.ridi.io/store/store.html#operation/GET_user_book-download
http://api.dev.ridi.io/platform/download.html#operation/GET_user_book-download

API 명세는 위 두가지 보시면 되고요 POST 는 아직 배포 전이라 없을거에요.

Reply:
백엔드 문의결과
이미 검증된 b_id 를 해당 endpoint 를 사용해서 다시 받아오는 과정이 불필요 하기 때문에 앱 스킴만 받는 별도의 endpoint 를 만들어 사용하는게 나을 것 같고 CS의 빠른 대응을 위해 POST를 추가한 것인데 이와 같은 상황에서 잘못된 content-type 이 아님에도 json 대응하는 작업을 해야하는 데에 대한 부담이 있다는 답변을 받았습니다.

Reply:
으으으음 빠른 대응이 필요하다면 잘 작동한다는 가정 하에 우선 이대로 나가도 될 것 같은데요, 배열을 urlencoded로 보내는 건 정해진 표준이 없어서 좀 불안하네요

Reply:
해당 문법은 PHP에서 지원하는 것이긴 한데, 프론트엔드 팀이 분리된 이상 특정 백엔드 상황을 가정하여 코드를 
작성하는 것은 좋지 않은 듯 합니다. 😅

@nnyamm 관련된 논의 링크를 알려주실 수 있을까요? 요청의 Content-Type 헤더가 application/json 인 경우 기존 컨트롤러에서 json decode만 해주면 될 것 같아서요.

Reply:
@namenu 
태스크는 다음과 같고요 https://app.asana.com/0/8314863450537/1131217457788471 
신순님과 구두로 Content-Type 헤더 변경 관련한 요청을 드리며 나눈 내용을 코멘트로 작성 한 것입니다.